### PR TITLE
Handle looping for Android

### PR DESCRIPTION
**Pull request by pawlowskim**
Hello, I needed looping on Android to works the same way as for iOS. I changed current onCompletionListener with custom one, that handles loops. Api is the same for iOS and Android. I'm not java master so any suggestion will be appreciated.

Sorry for formatting :/

### CODE REVIEW

**Comment by benvium** [comment]
@pawlowskim Thanks for this. Could you try to apply your changes again without all the formatting updates? It's difficult to keep track of changes if the whole file is reformatted

**Comment by pawlowskim** [comment]
@benvium Second commit should keep old formatting.

File: android/src/main/java/com/zmxv/RNSound/RNSoundModule.java [hunk 1]
```diff
@@ -258,3 +255,39 @@ public void enable(final Boolean enabled) {
     return constants;
   }
 }
+
+class OnCompletionListenerLooped implements OnCompletionListener {
```
Comment (benvium) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Probably best to be `static class OnCompletionListenerLooped`

Comment (pawlowskim) [inline android/src/main/java/com/zmxv/RNSound/RNSoundModule.java]: Why static? Nobody gonna use it outside the RNSoundModule and static is little heavier than private. Isn't it? I'm not java master.

**Comment by paulmelnikow** [comment]
Hi! Thank you for this! If you'd like to rebase this work onto latest master I'd be happy to get it merged! 😀

**Comment by pawlowskim** [comment]
@paulmelnikow This PR is too old. I have created new one: https://github.com/zmxv/react-native-sound/pull/586

I will close this.

**Pull request closed without merging**